src/dialogs/connectsettingsdialog.cpp

```diff
@@ -38,3 +39,22 @@ void ConnectSettingsDialog::locateSerialPort()
         ui->serialPortCB->clear();
     }
 }
+
+void ConnectSettingsDialog::initFirmwareComboBox()
+{
+    QStringList fw;
+    fw.append("Auto-Detect");
+    fw.append("Marlin");
+    fw.append("Repetier");
+    fw.append("Sprinter");
+    fw.append("Teacup");
+    ui->firmwareCB->addItems(fw);
```

Comment:
Yeah, that part was a doubt... But how can I get the list from the namespace? The correct is only display the fw that we have support. The namespace have a list of all RepRap firmwares, but we don't have support for all. What is the best? Remove the ones that we don't have support?